<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="NGBLR-00763" name="Validate Title" tests="2" failures="2" timestamp="28 Dec 2020 21:45:17 GMT" time="6.514" errors="0">
  <testcase name="BaseNavigation" time="0.083" classname="Reports.ValidatingTitle">
    <failure type="java.lang.AssertionError" message="expected [FetauredCourses] but found [FEATURED COURSES]">
      <![CDATA[java.lang.AssertionError: expected [FetauredCourses] but found [FEATURED COURSES]
	at Reports.ValidatingTitle.BaseNavigation(ValidatingTitle.java:37)
	at java.util.concurrent.FutureTask.run(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
... Removed 22 stack frames]]>
    </failure>
  </testcase> <!-- BaseNavigation -->
  <testcase name="BaseNavigation" time="0.092" classname="Reports.ValidatingTitle">
    <failure type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
	at resources.Base.getScreenshotPath(Base.java:49)
	at Reports.Listeners.onTestFailure(Listeners.java:45)
	at java.util.concurrent.FutureTask.run(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
... Removed 13 stack frames]]>
    </failure>
  </testcase> <!-- BaseNavigation -->
</testsuite> <!-- Validate Title -->
